lib/preparser.ts

```diff
@@ -133,6 +133,8 @@ export const convertedCommands = [
 	'env:rm',
 	'os:configure',
 	'version',
+	'settings',
+	'note',
```

Comment:
The list was in alphabetical order until this PR. :-)  (The CLI has some 60 commands/subcommands, eventually alphabetical order will make it easier to find something in the list.)